### PR TITLE
:mortar_board: Topic introduction --- Make topic

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Write the introduction topic

### Why

Feels weird not having a topic as a quick overview of WTF the course is about. 

### How

1. Quick and high-level. The point is to give them an idea of what the course is about
2. There are several of images to try to make things look exciting

### Testing

:+1: 

### Additional Notes

Ben Eater is the GOAT

Also, this topic will likely change a little once I actually have all the course content in place.

### CODE REVIEW

File: site/topics/introduction/introduction.rst [hunk 1]
```diff
@@ -2,12 +2,112 @@
...
+    :target: https://en.wikipedia.org/wiki/Von_Neumann_architecture
+
+    High-level idea of the von Neumann architecture. One of the interesting features of this architecture is how memory
+    stores both data and instructions.
+
+
+.. figure:: harvard_architecture.png
+    :width: 500 px
+    :align: center
+    :target: https://en.wikipedia.org/wiki/Von_Neumann_architecture
```
Comment (twentylemon) [inline site/topics/introduction/introduction.rst]: wrong link

File: site/topics/introduction/introduction.rst [hunk 2]
```diff
@@ -2,12 +2,112 @@
...
+    Screenshot of an ALU from a larger computer made within the game "Turing Complete".
+
 
 
 For Next Time
 =============
 
 * Get your computer at home :doc:`set up for the course </getting-set/getting-set>`
+* `Watch Ben Eater's 8-bit computer overview video <https://www.youtube.com/watch?v=HyznrdDSSGM&list=PLowKtXNTBypGqImE405J2565dvjafglHU&index=1>`_
+* Play the level "Crude Awakening" from the *Turing Complete* game
```
Comment (twentylemon) [inline site/topics/introduction/introduction.rst:132]: _play a game_ -- think it's a bit unreasonable to ask students to buy a game. Mentioning the game is one thing, but giving homework to play it feels off. Potentially legally questionable too.

Comment (jameshughes89) [reply site/topics/introduction/introduction.rst:132]: Nah. Same deal as textbooks. Any resources. Besides, it's not required.

Comment (twentylemon) [reply site/topics/introduction/introduction.rst:132]: That is not made clear. The additional resources section starts off with a required resource, then lists these optional ones. Then work is assigned from the optional ones. Make it make sense.

File: site/topics/introduction/introduction.rst [hunk 3]
```diff
@@ -77,6 +77,28 @@ Additional Resources
...
 
+Supplementary Resources
+-----------------------
+
+* This course has several recommended supplementary resources
+
+    * Not required, but provide additional practice, context, details, etc.
+
+
+* At tne end of each topic, content from various resources will be suggested
```
Comment (twentylemon) [inline site/topics/introduction/introduction.rst:88]: typo _the_

**Review by twentylemon — APPROVED** [review]

**Pull request merged**